docs/build-from-scratch.md

```diff
@@ -21,6 +21,10 @@ The files are installed in `$GOPATH/src/github.com/heptio/ark`.
 
 ## 2. Build
 
+### Using a locally built image
+
+To ensure that your Ark deployment uses your local image rather than pulling from an image registry, you'll need to explicitly set the `spec.template.spec.containers[*].imagePullPolicy` in your deployment file to `IfNotPresent`. This file is either `examples/azure/00-ark-deployment.yaml` if you're using Azure, or `examples/common/10-deployment.yaml` for all other cloud providers.
+
 ### Using your own registry
 
 If you want to push the Heptio Ark images to your own registry, set the `$REGISTRY` environment variable (used in the `Makefile`). This allows any node in your cluster to pull your locally built image.
```

Comment:
Just realized I think there's a problem with this too - $REGISTRY is also used to locate the build image (which is in heptio-images), so changing it to a personal registry will break the make if they don't have the builder image.